pliers/extractors/text.py

```diff
@@ -402,3 +400,140 @@ def _extract(self, stim):
 
         return ExtractorResult(features_list, stim, self,
                                features=self.features, orders=order_list)
+
+
+class PretrainedBertEncodingExtractor(ComplexTextExtractor):
+
+    ''' Uses transformers library to extract contextualized encodings for words
+    or text sequences using pre-trained BERT models.
+
+    Args:
+        pretrained_model_or_path (str): A string specifying which BERT
+            model to use. Can be one of pretrained BERT models listed at
+            https://huggingface.co/transformers/pretrained_models.html
+            or path to custom model.
+        tokenizer (str): Type of tokenization used in the tokenization step.
+            If different from model, out-of-vocabulary tokens may be treated as
+            unknown tokens.
+        framework (str): name deep learning framework to use. Must be 'pt'
```

Comment:
pedantic, but for consistency, maybe go with either the values "torch" and "tensorflow" or the values "pt" and "tf".

Reply:
fixed